app/view/addCompany/addCompany.html

```diff
@@ -1 +1,26 @@
-<p>This is the partial for adding a new company.</p>
+<div class="row">
+  <div class="col-12 col-lg-8">
+    <h1>Add a new company</h1>
+
+    <form>
+      <div class="form-group">
+        <label for="company_name">What's the name of the company you want to add?</label>
+        <input class="form-control" id="company_name" ng-model="company_name" placeholder="Name" required>
```

Comment:
<input class="form-control" id="company_name" ng-model="company.name" placeholder="Name" required>

Reply:
Something's wrong with your comments dude. Do you see something here? For me it's just a blank comment.